src/api/routes.py

```diff
@@ -46,7 +56,13 @@ async def response_stream():
         if globals["rag"] is not None:
             context = await globals["rag"].search(chat_request)
             if context:
+                # Clean up the context to avoid non unicode characters.
+                context = context.encode("ascii", "ignore").decode("utf-8")
```

Comment:
would this work?

context = context.encode("utf-8", errors="ignore").decode("utf-8")

Reply:
Or create bug to search team if the input is valid utf-8

Reply:
Removed this code as it is not needed.